### PR TITLE
msvc debug: fix wrong lib name in generated pkgconfig file

### PR DESCRIPTION
**Pull request by autoantwort**
Otherwise the d postfix is missing in the DEBUG build on windows in the generated pkgconf file

### CODE REVIEW

**Comment by DanBloomberg** [comment]
Thank  you for this PR.   Someone who understands the cmake system and msvc will look at it.

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -19,13 +19,8 @@ string(REPLACE "-O3" "-O2" CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE}")
...
 set_target_properties           (leptonica PROPERTIES SOVERSION 6)
-if (WIN32)
-set_target_properties           (leptonica PROPERTIES OUTPUT_NAME       leptonica-${VERSION_PLAIN})
-set_target_properties           (leptonica PROPERTIES DEBUG_OUTPUT_NAME leptonica-${VERSION_PLAIN}d)
-else()
-set_target_properties           (leptonica PROPERTIES OUTPUT_NAME       leptonica)
-set_target_properties           (leptonica PROPERTIES DEBUG_OUTPUT_NAME leptonica)
-endif()
+
+set_target_properties           (leptonica PROPERTIES OUTPUT_NAME tesseract$<$<BOOL:${WIN32}>:-${VERSION_PLAIN}$<$<CONFIG:DEBUG>:d>>)
```
Comment (autoantwort) [inline src/CMakeLists.txt:23]:
```suggestion
set_target_properties           (leptonica PROPERTIES OUTPUT_NAME leptonica$<$<BOOL:${WIN32}>:-${VERSION_PLAIN}$<$<CONFIG:DEBUG>:d>>)
```
Oups copy paste error

**Comment by zdenop** [comment]
Dan, it is ok.

**Comment by DanBloomberg** [comment]
Merged.

**Comment by DanBloomberg** [comment]
We;'ll find out -- I forgot to run the workflow !!!

**Comment by DanBloomberg** [comment]
and #665 has conflicts with this one

**Pull request merged**